### DISCUSSION TITLE
How to avoid multiple posting actions using wire:click

### DISCUSSION THREAD

**Discussion by pleungkh** [livewire/livewire#2029] [1. Help]
I just started a brand new tiny app with Laravel 8 / livewire 2.0

In my Livewire view component, postLoadJob was call via wire:click.
 <button  wire:click="postLoadJob()"
In my Livewire Server class
    public function postLoadJob() {
...
      // it executes some slow processes to access remote server via ssh with laravel-collective remote 

...
   }  

The above button posting works well. However,  the button posting returns very slow ( 4 - 5 seconds).
This a is bad user experience that users may click several times and fire multiple posting actions.

Does anyone know how to avoid that?

**Comment by joshhanley** [comment]
You can use wire:loading to disable the button, check it out here https://laravel-livewire.com/docs/2.x/loading-states

Hope this helps!

**Reply by pleungkh** [reply]
It works!  Thanks joshhanley!
